docs/tutorials/typescript/getting_started.md

```diff
@@ -0,0 +1,141 @@
+# NEAR AI Typescript SDK - Getting Started
+
+## Prerequisites
+
+Before creating a NEAR AI Typescript Agent, please make sure you have the [NEAR AI CLI](https://docs.near.ai/cli/) installed and have logged in with your Near wallet.
+
+- [ ] You have `Python 3.11.xx` in your machine. (**IMPORTANT** `Python 3.12+` currently not supported!)
+- [ ] Installing NEAR AI CLI `python3 -m pip install nearai`
+- [ ] Login to NEAR AI `nearai login`
+
+> You'll be provided with a URL to login with your NEAR account.
+>
+> Example:
+
+```shell
+$> nearai login
+
+Please visit the following URL to complete the login process: https://auth.near.ai?message=Welcome+to+NEAR+AI&nonce=<xyzxyzxyzxyzx>&recipient=ai.near&callbackUrl=http%3A%2F%2Flocalhost%3A63130%2Fcapture
+```
+
+- [ ] Login with NEAR by connecting with your wallet. After successfully logging in, you will see a confirmation screen. Close it and return to your terminal.
+
+## Local Development
+
+Clone the NEAR AI Github Repository
+
+```shell
+git clone https://github.com/nearai/nearai.git
+```
+
+Change your directory to `ts_runner/ts_agent` and Install `Node.js` dependencies by running the command below
+
+```shell
+cd ts_runner/ts_agent && npm install
+```
+
+Change the directory to `ts_runner/ts_agent/agents` and add your agent related code in `agent.ts` file
+
+```typescript
+import { env } from "ts-agent-runner";
+
+(async () => {
+  try {
+    const userMessage = "Tell me an AI joke";
+
+    console.log("User input:", userMessage);
+
+    const messages: any = [
+      {
+        role: "system",
+        content: "You are a smart assistant.",
+      },
+      {
+        role: "user",
+        content: userMessage,
+      },
+    ];
+
+    // inference
+    const reply = await env.completion(
+      messages,
+      "llama-v3p1-70b-instruct",
+      4000,
+      0.5
+    );
+
+    if (env.get_thread_id() !== "thread_local") {
+      await env.add_reply(reply);
+    }
+
+    console.log("Agent output:", reply);
+  } catch (error) {
+    console.error("Agent error:", error);
+  }
+})();
+```
+
+Run the agent
```

Comment:
Running the agent just printed out a joke and then shut down the agent, was that the intention? I would expect it to run like our other agents where the user can continue to interact with it until you exit out.